utils/http_utils.go

```diff
@@ -49,6 +49,10 @@ func HttpExecute(httpClient *http.Client, logger *log.Logger, requestType string
 		return nil, fmt.Errorf("Error in creating request")
 	}
 
+        request.Header.Add("Content-Type","application/json")
+        request.Header.Add("Accept","application/json")
+
+        request.SetBasicAuth(user,password)
```

Comment:
add a check if these params are specified or not (they might be empty if called from remote/client package)